### PR TITLE
Issue/1349 mvv card settings

### PR DESCRIPTION
**Pull request by Liqs-v2**
## Issue

This fixes the following issue(s): When the preferred stop for a campus location is changed, the change is not reflected in the MVVCard that is displayed in the app's homescreen.

### Broken version
https://user-images.githubusercontent.com/9247500/144461001-e2ba6fb2-0ecb-4dc8-953b-e0ced3d184c3.mp4

### Fixed version
https://user-images.githubusercontent.com/9247500/144458358-51182d5b-5006-4981-8743-3688121e6688.mp4

## Followup suggestion
Changing the default campus location also seems broken in regards to the MVVCard. I'll look into this. **This has been fixed, see below.**

## Other
It seems as though the [incorrect namespace for preference attributes was used](https://developer.android.com/guide/topics/ui/settings/) for the settings. Everything seems to still work, but I find this to be confusing. Perhaps some clarification would be good.

## Why this is useful for all students
Being able to see public transport departures for the public transport station that one is actually going to use, is a lot more useful, than just getting information regarding departures for the default stationthat was selected by the developers for a specific campus.

### CODE REVIEW

**Comment by Liqs-v2** [comment]
Changing the default campus requires a restart currently for the MVVCard to be updated.

**Comment by Liqs-v2** [comment]
Changing the default campus no longer breaks the MVVCard now:

### Previous verion
https://user-images.githubusercontent.com/9247500/144565667-2cecd22e-bdd7-458d-9036-4d8327a18354.mp4



### Fixed version
https://user-images.githubusercontent.com/9247500/144565691-de105f14-185a-41de-8337-6597355cc26f.mp4

File: app/src/main/res/xml/settings.xml [hunk 1]
```diff
@@ -380,7 +380,7 @@
             android:key="employee_mode"
             android:summary="@string/employee_mode_info"
             android:title="@string/employee_mode" />
-        <androidx.preference.Preference
+        <androidx.preference.Preferenced
```
Comment (joschahenningsen) [inline app/src/main/res/xml/settings.xml:383]:
```suggestion
        <androidx.preference.Preference
```

Comment (Liqs-v2) [reply app/src/main/res/xml/settings.xml]: This seems to be a type, fixed that.

**Review by joschahenningsen — APPROVED** [review]
Thanks :)

**Pull request merged**